tslearn/clustering.py

```diff
@@ -704,10 +704,13 @@ def _assign(self, X):
         _check_no_empty_cluster(self.labels_, self.n_clusters)
         self.inertia_ = _compute_inertia(dists, self.labels_)
 
-    def _fit_one_init(self, X, rs):
+    def _fit_one_init(self, X, initial_centroids, rs):
```

Comment:
Would it be possible to rename this argument as `init` and more or less match sklearn usage (cf. https://github.com/scikit-learn/scikit-learn/blob/f0ab589f/sklearn/cluster/k_means_.py#L707)?